### DISCUSSION TITLE
1.32.0

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#18165] [General]
# Notes
- For CC2652/CC1352 users: a new firmware is available which should improve performance ([`20230507`](https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin)).

# Improvements
- [#5928](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5928) Add additional `pressed` action for HEIMAN HS2SS-E_V03 (@Anaethelion)
- [#5914](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5914) Support occupancy and illuminance for Third Reality 3RSNL02043Z (@llhappier)
- [#11483](https://github.com/Koenkk/zigbee2mqtt/issues/11483) Support child lock for TuYa TS011F_2_gang_2_usb_wall  (@rodrigogbs)
- [#5899](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5899) Support OTA for Legrand 067771 and 067772 (@EXP-Carthage)
- [#17977](https://github.com/Koenkk/zigbee2mqtt/issues/17977) Allow to set more sensitivity levels for TuYa TS0210 (@Koenkk)
- [#325](https://github.com/Koenkk/zigbee-OTA/pull/325) Support OTA for Moes MS-108ZR  (@cserem)
- [#17970](https://github.com/Koenkk/zigbee2mqtt/issues/17970) Support `power_on_behavior` for Gledopto GL-B-007P (@Koenkk)
- [#5856](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5856) Expose power per phase and add desriptions for TuYa TS0601_rcbo (@octa22)
- [#5848](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5848) Expose `MOTDETAT` for Lixee ZiPulses (@SilentT-FR)
- [#318](https://github.com/Koenkk/zigbee-OTA/pull/318) Support OTA for NodOn SIN-4-RS-20  (@AlexisPolegato)
- [#17856](https://github.com/Koenkk/zigbee2mqtt/issues/17856) Make lookup commands to TuYa devices case insensitive (@Koenkk)
- [#727](https://github.com/Koenkk/zigbee-herdsman/issues/727) Remove EZSP queue to improve message throughput (@kirovilya)
- [#718](https://github.com/Koenkk/zigbee-herdsman/issues/718) Improve request queue behavior (@slugzero)

# Fixes
- [#5934](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5934) Fix naming inconsistency for vendor Nous (@pomeloy)
- [#5925](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5925) Change brightness min reporting to 1 to prevent spamming of some devices (@monkeyclass)
- [#5931](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5931) Fix naming inconsistency for vendor LED-Trading (@pomeloy)
- [#18088](https://github.com/Koenkk/zigbee2mqtt/issues/18088) Fix actions missing for PTM 215Z detected as Hue tap (@Koenkk)
- [#18107](https://github.com/Koenkk/zigbee2mqtt/issues/18107) Fix EWeLink ZB-SW02 description (@Koenkk)
- [#5905](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5905) Fix MULTI-ZIG-SW not working (@mortenmoulder)
- [#18041](https://github.com/Koenkk/zigbee2mqtt/issues/18041) Fix TuYa TS000F_power current value incorrect (@Koenkk)
- [#2095](https://github.com/Koenkk/zigbee2mqtt.io/pull/2095) Move 99099 from AwoX to EGLO (@Koenkk)
- [#5892](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5892) Fix invalid thermostat value handling (@tomasbedrich)
- [#5894](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5894) Fix `ptvo.switch` not working (@ptvoinfo)
- [#5891](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5891) Rename ZY-M100-S to ZY-M100-L and TS0601_smart_human_presence_sensor_2 to ZY-M100-S (@pswid)
- [#5897](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5897) RTX ZVG1 water_consumed unit (l -&gt; L) (@kamaradclimber)
- [#5886](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5886) Fix Lidl HG08164 disconnecting from network (@Koenkk)
- [#18007](https://github.com/Koenkk/zigbee2mqtt/issues/18007) Fix LifeControl MCLH-04 battery % incorrect (@Koenkk)
- [#14923](https://github.com/Koenkk/zigbee2mqtt/issues/14923) Disable OTA for Philips 9290030674 as none is available currently (@Koenkk)
- [#14923](https://github.com/Koenkk/zigbee2mqtt/issues/14923) Disable OTA for Philips 9290030675 since non is available (@Koenkk)
- [#5871](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5871) Normalise Mercator Ikuü vendor and device naming (@davidjb)
- [#2078](https://github.com/Koenkk/zigbee2mqtt.io/pull/2078) Disable unsupported `power_on_behavior` for LEDVANCE 4058075729322  (@GerdRuetten)
- [#317](https://github.com/Koenkk/zigbee-OTA/pull/317) Fix Ubisys C4, D1, J1, R0, S2 and S1-R latest OTA not available  (@WhistleMaster)
- [#5857](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5857) Fix strange "Â" in device description and unit (@Koenkk)
- [#5842](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5842) Disable unsupported `power_on_behaviour` for Innr FL 130 C (@weltmeyer)
- [#17883](https://github.com/Koenkk/zigbee2mqtt/issues/17883) Fix TuYa TS011F_1 and TS011F_4 not detected (@Koenkk)
- [#5811](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5811) Align options.supportsHS with meta.supportsHueAndSaturation (@sjorge)
- [#5820](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5820) Fix HSV colors incorrect (disable gamma correction) (@Zoooook)
- [#5830](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5830) Fix TuYa TS0601_dimmer_knob `indicator_mode` (@Koenkk)
- [#726](https://github.com/Koenkk/zigbee-herdsman/issues/726) Set correct direction for default response. https://github.com/Koenkk/zigbee2mqtt/issues/18096 (@Koenkk)
- [#721](https://github.com/Koenkk/zigbee-herdsman/issues/721) Fix EZSP group binding (@tomasbedrich)

# New supported devices
This release adds support for 39 devices: 
- [#5917](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5917) `4058075729162` LEDVANCE SMART+ PAR16 GU10 Tunable white (@dhanukapulkit)
- [#5917](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5917) `4058075729087` LEDVANCE SMART+ CL B40 E14 Tunable white (@dhanukapulkit)
- [#5917](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5917) `4058075729346` LEDVANCE SMART+ Gardenpole 5P Multicolor (@dhanukapulkit)
- [#5927](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5927) `SWS6TZ-WHITE` TuYa 6 gang wall switch (@nuwan-perera-au)
- [#5916](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5916) `03982` Vimar Roller shutter with slat orientation and change-over relay (@giovannismart)
- [#5915](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5915) `BTH-RM` Bosch Room thermostat II (@dennismaxjung)
- [#5902](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5902) `Hive` Phoscon Battery powered smart LED light (@Backup7750)
- [#18111](https://github.com/Koenkk/zigbee2mqtt/issues/18111) `BD05C-FL-21-G-ENK` ADEO ENKI RGBCCT lamp (@Koenkk)
- [#5909](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5909) `ZNQBKG25LM` Xiaomi Aqara smart wall switch H1M (with neutral, double rocker) (@Whipsmith)
- [#5904](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5904) `U86Z13A16-ZJH(HA)` Honyar Smart Power Socket 16A (with power monitoring) (@guotianx)
- [#18081](https://github.com/Koenkk/zigbee2mqtt/issues/18081) `C202` Candeo Zigbee LED smart dimmer switch (@Koenkk)
- [#18087](https://github.com/Koenkk/zigbee2mqtt/issues/18087) `ST8EM-CON` LEDVANCE SubstiTUBE connected advanced ultra output (@saschaludwig)
- [#18086](https://github.com/Koenkk/zigbee2mqtt/issues/18086) `TERNCY-WS01` TERNCY Smart light switch - 4 gang without neutral wire (@gcmilo704)
- [#5895](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5895) `4512749-N` Namron Thermostat outlet socket (@danielfinans)
- [#5893](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5893) `CL001` TERNCY Beevon ceiling light (@Qianli-Ma)
- [#5883](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5883) `TS0001_fingerbot` TuYa Zigbee fingerbot plus (@kirovilya)
- [#5889](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5889) `TS0002_switch_module_3` TuYa 2 gang switch with backlight (@dimich99)
- [#5884](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5884) `929003116101` Philips Hue Perifo linear light bar (black) (@StevenReitsma)
- [#5884](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5884) `929003115701` Philips Hue Perifo cylinder spotlight (black) (@StevenReitsma)
- [#5888](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5888) `98847` EGLO FUEVA-Z ceiling light IP44 (@pswid)
- [#5877](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5877) `MULTI-ZIG-SW` smarthjemmet.dk [Multi switch from Smarthjemmet.dk](https://smarthjemmet.dk) (@mortenmoulder)
- [#17978](https://github.com/Koenkk/zigbee2mqtt/issues/17978) `RB 255 C` Innr E14 mini bulb RGBW (@B08Z)
- [#5875](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5875) `3RSNL02043Z` Third Reality Zigbee multi-function night light (@llhappier)
- [#17981](https://github.com/Koenkk/zigbee2mqtt/issues/17981) `929003128401` Philips Hue Go portable table lamp white (@KHOne23)
- [#5876](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5876) `968.93` Paulmann URail rail adapter smart home Zigbee on/off/dimm (@gritche2)
- [#5868](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5868) `SIN-4-1-20_LEX` ADEO ENKI LEXMAN 3680W single output relay (@ScratMan)
- [#5859](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5859) `D077-ZG` HZC Zigbee dimmer (@Koenkk)
- [#17965](https://github.com/Koenkk/zigbee2mqtt/issues/17965) `OFL 122 C` Innr Outdoor flex light colour LED strip 2m, 1440lm, RGBW (@Koenkk)
- [#5863](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5863) `ZNQBKG24LM` Xiaomi Aqara smart wall switch H1M (with neutral, single rocker) (@z-x-f)
- [#5854](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5854) `929002401001` Philips Hue Iris gold special edition (generation 4) (@ebarajas)
- [#17933](https://github.com/Koenkk/zigbee2mqtt/issues/17933) `948.47` Paulmann RGBW wall light (@fsedarkalex)
- [#5853](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5853) `99099` EGLO 3 groups remote controller (@fsedarkalex)
- [#17932](https://github.com/Koenkk/zigbee2mqtt/issues/17932) `12226` AwoX Dimmable filament lamp (@fsedarkalex)
- [#17933](https://github.com/Koenkk/zigbee2mqtt/issues/17933) `948.47` Paulmann RGBW wall light (@fsedarkalex)
- [#17890](https://github.com/Koenkk/zigbee2mqtt/issues/17890) `929003128701` Philips Hue Go portable table lamp (@Koenkk)
- [#5835](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5835) `TH-110-ZB` Dawon DNS IoT SMART temperature and humidity sensor (@nukusinji)
- [#17840](https://github.com/Koenkk/zigbee2mqtt/issues/17840) `HA-ZGMW2-E` Halemeier LED driver (@Koenkk)
- [#5827](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5827) `ZSS-QY-SSD-A-EN` TuYa Smart smoke alarm (@Luligu)
- [#18130](https://github.com/Koenkk/zigbee2mqtt/issues/18130) `TS0601_bidirectional_energy meter` TuYa Bidirectional energy meter with 150A Current Clamp (@fred-c1)

# Fixed device detections
- [#17979](https://github.com/Koenkk/zigbee2mqtt/issues/17979) Detect `_TZE200_bv1jcqqu` as Zemismart ZM25RX-08/30  (@tekman54190)
- [#5923](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5923) Detect `SV01-612-MP-1.4` as Keen Home SV01 (@TomaszKorwel)
- [#5911](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5911) Detect `_TZE200_ga1maeof` as TuYa TS0601_soil (@Slaw3km)
- [#18102](https://github.com/Koenkk/zigbee2mqtt/issues/18102) Detect `_TZ3000_u3oupgdy` as MHCOZY TYWB 4ch-RF (@Koenkk)
- [#18098](https://github.com/Koenkk/zigbee2mqtt/issues/18098) Detect `_TZ3000_0s1izerx` as TuYa ZTH01/ZTH02 (@Koenkk)
- [#18097](https://github.com/Koenkk/zigbee2mqtt/issues/18097) Detect `_TZE204_aoclfnxz` as Moes BHT-006GBZB (@Koenkk)
- [#5900](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5900) Detect `_TZE200_qtbrwrfv` as Alecto SMART-SMOKE10 (@Koenkk)
- [#18057](https://github.com/Koenkk/zigbee2mqtt/issues/18057) Detect `_TZE200_44af8vyi` as TuYa TS0601_temperature_humidity_sensor_1 (@Koenkk)
- [#5881](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5881) Detect `_TZ3000_g92baclx` as TuYa TS0001_power (@aldarund)
- [#18016](https://github.com/Koenkk/zigbee2mqtt/issues/18016) Detect `_TZE200_axgvo9jh` as Somgoms ZSTY-SM-1DMZG-US-W_1 (@Koenkk)
- [#18024](https://github.com/Koenkk/zigbee2mqtt/issues/18024) Detect `HK_DIM_A` as Candeo HK-DIM-A  (@maans2001)
- [#5879](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5879) Detect `_TZ3000_rqbjepe8` as Nous A4Z (@Koenkk)
- [#17969](https://github.com/Koenkk/zigbee2mqtt/issues/17969) Detect `_TZE200_leaqthqq` as TuYa TS0601_switch_5_gang (@Koenkk)
- [#17996](https://github.com/Koenkk/zigbee2mqtt/issues/17996) Detect `_TZ3000_ss98ec5d` as Moes ZK-EU (@Koenkk)
- [#5870](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5870) Detect `_TZ3000_zw7wr5uo` as Mercator Ikuü SMI7040 (@davidjb)
- [#17973](https://github.com/Koenkk/zigbee2mqtt/discussions/17973) Detect `_TZE200_nyvavzbj` as Immax 07505L (@Koenkk)
- [#5864](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5864) Detect `_TZ3000_lepzuhto` as EARU EAKCB-T-M-Z (@M66B)
- [#17937](https://github.com/Koenkk/zigbee2mqtt/issues/17937) Detect `SM323` as Samotech SM323 (@Koenkk)
- [#17928](https://github.com/Koenkk/zigbee2mqtt/issues/17928) Detect `_TZ3000_nnwehhst` as TuYa TS0003_switch_module_1  (@mersadk)
- [#5851](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5851) Detect `_TZE200_bjzrowv2` as TuYa TS0601_cover_1 (@kclif9)
- [#5849](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5849) Detect `RDM003` as Philips 8718699693985 (@samthor)
- [#17913](https://github.com/Koenkk/zigbee2mqtt/issues/17913) Detect `_TZE200_9cxuhakf` as Mercator Ikuü SSWM-DIMZ (@Koenkk)
- [#5840](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5840) Detect `_TZE200_bqcqqjpb` as Yushun YS-MT750L (@at-9)
- [#17439](https://github.com/Koenkk/zigbee2mqtt/issues/17439) Detect `_TZE200_mja3fuja` as TuYa TS0601_smart_air_house_keeper (@Koenkk)
- [#17907](https://github.com/Koenkk/zigbee2mqtt/issues/17907) Detect `_TZ3000_4rbqgcuv` as AVATTO ZWSM16-1-Zigbee (@Koenkk)
- [#17902](https://github.com/Koenkk/zigbee2mqtt/issues/17902) Detect `_TZ3000_mtnpt6ws` as AVATTO ZWSM16-2-Zigbee (@Koenkk)
- [#11251](https://github.com/Koenkk/zigbee2mqtt/issues/11251) Detect `_TZE200_nw1r9hp6` as Zemismart ZM85EL-2Z (@Koenkk)
- [#5834](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5834) Detect `_TZE204_wbhaespm` as RTX ZCR1-40EM (@sgurgul)
- [#17857](https://github.com/Koenkk/zigbee2mqtt/issues/17857) Detect `_TZE204_5toc8efa` as Moes BHT-002-GCLZB (@Koenkk)
- [#5878](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5878) Detect `_TZE200_nlrfgpny` as Neo NAS-AB06B2 (@tygroo)

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.32.0'>1.32.0</a>.</em>

**Comment by lux73** [comment]
upgrade plain z2m form 1.31.0 -> 1.32.0 went smooth

Nice, Thanks!

**Comment by Nardol** [comment]
Since I updated to 1.32.0 I repeatedly have this kind of error in the log: "Exception while calling fromZigbee converter: 'energy_calibration' is not a number, got string ()}.
I opened issue #18169 about this issue.

**Comment by palsbo** [comment]
### **Problem** v1.31.2 -> v1.32.0
In version 1.31.2 a device was registred as Model='**Eco-Dim.07/Eco-Dim.10**', Producer='**EcoDim**'
in version 1.32.0 it was registred as Model='**Dimmer-Switch-ZB3.0**', Producer=**Not Supported**!!!
What to do to get it back and suppoerted?
